docs/index.md

```diff
@@ -17,16 +17,24 @@ Pebble is useful for developers who are building [Juju charms on Kubernetes](htt
 
 ```{grid-item-card} [How-to guides](how-to/index)
 **Step-by-step guides** covering key operations and common tasks.
+- [Install Pebble](how-to/install-pebble)
+- [Manage service dependencies](how-to/service-dependencies)
+- [Manage identities](how-to/manage-identities)
 ```
 ````
 
 ````{grid} 1 1 2 2
 ```{grid-item-card} [Explanation](explanation/index)
 **Discussion and clarification** of key topics.
```

Comment:
This probably shouldn't have a `.` at the end, as it's not a full sentence. Same with the other subtitles.